src/components/Web3Provider/index.test.tsx

```diff
@@ -36,9 +37,11 @@ function last<T>(array: T[]): T {
 }
 
 const UI = (
-  <Provider store={store}>
-    <Web3Provider>{null}</Web3Provider>
-  </Provider>
+  <HashRouter>
```

Comment:
`useLocation` causes test to fail unless i wrap in a Router component

Reply:
can you use the `test-utils/render` function instead?